package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "CDMX009-MdLinks",
+  "version": "1.0.0",
+  "description": "## Preámbulo",
```

tu descripción tiene que ser funcional y en ingles
```suggestion
  "description": "## Preámbulo",
```